src/index.js

```diff
@@ -0,0 +1,204 @@
+};
+// Update Local Storage
+const sendToLocalAgain = () => {
+  const localData = JSON.parse(localStorage.getItem('list'));
+  const todos = document.querySelectorAll('span');
+
+  for (let i = 0; i < todos.length; i += 1) {
+    if (todos[i].classList.contains('checkTodo')) {
+      localData[i].completed = true;
+    } else {
+      localData[i].completed = false;
+    }
+  }
+
+  localStorage.setItem('list', JSON.stringify(localData));
+};
+
+const getTasksFromLocStore = () => {
+  const data = JSON.parse(localStorage.getItem('list'));
+  // eslint-disable-next-line array-callback-return
+  data.map((tasks) => {
+    myArray.push(tasks);
+    const todoContainer = document.createElement('div');
+    todoContainer.className = 'todoContainer';
+    todoContainer.innerHTML += `
+        <input type='checkbox' class='checkbox'>
+        <span>${tasks.description}</span>
+        <i class="uil uil-ellipsis-v"></i>
+        <i class="uil uil-trash-alt"></i>
+    `;
+    todosParent.appendChild(todoContainer);
+
+    const editIcons = document.querySelectorAll('.uil-ellipsis-v');
+    editIcons.forEach((task) => {
+      task.addEventListener('click', () => {
+        editTodo(todoContainer, task.previousElementSibling);
+        task.parentElement.classList.add('completedTask');
+      });
+    });
+  });
+
+  const checkbox = document.querySelectorAll('.checkbox');
+  checkbox.forEach((task) => {
+    task.addEventListener('click', () => {
+      task.parentElement.classList.toggle('completedTask');
+      task.nextElementSibling.classList.toggle('checkTodo');
+      task.parentElement.lastElementChild.classList.toggle('trash-active');
+      task.parentElement.lastElementChild.previousElementSibling.classList.toggle('edit-disable');
+      sendToLocalAgain();
+    });
+  });
+
+  const removeIcons = document.querySelectorAll('.uil-trash-alt');
+  removeIcons.forEach((task) => {
+    task.addEventListener('click', () => {
+      deleteTask(task.parentElement);
+    });
+  });
+
+  localStorage.setItem('list', JSON.stringify(myArray));
+};
+
+// Add Tasks
+const addTask = (todoValue) => {
+  const todoContainer = document.createElement('div');
+  todoContainer.className = 'todoContainer';
+  todoContainer.innerHTML += `
+        <input type='checkbox' class='checkbox'>
+        <span>${todoValue}</span>
+        <i class="uil uil-ellipsis-v"></i>
+        <i class="uil uil-trash-alt"></i>
+    `;
+  todosParent.appendChild(todoContainer);
+
+  const checkbox = document.querySelectorAll('.checkbox');
+  checkbox.forEach((task) => {
+    task.addEventListener('click', () => {
+      task.parentElement.classList.toggle('completedTask');
+      task.nextElementSibling.classList.toggle('checkTodo');
+      task.parentElement.lastElementChild.classList.toggle('trash-active');
+      task.parentElement.lastElementChild.previousElementSibling.classList.toggle('edit-disable');
+      sendToLocalAgain();
+    });
+  });
```

Comment:
- [ ] When I click on the checkbox to mark a task as completed, it is marked as completed and the changes are saved in the local storage kudos for that 👏. **However, when I refresh the page the completed tasks are not reflected back again while they are marked as completed in the local storage.** In other words, when I refresh the page all the tasks even the completed ones appear as uncompleted while the ones that have been marked as completed should still appear as completed 😺 😉